internal_api.go

```diff
@@ -58,7 +58,7 @@ func (e *Enforcer) addPolicy(sec string, ptype string, rule []string) (bool, err
 	if e.watcher != nil && e.autoNotifyWatcher {
 		var err error
 		if watcher, ok := e.watcher.(persist.WatcherEx); ok {
-			err = watcher.UpdateForAddPolicy(rule...)
```

Comment:
@gonsuke I suggest adding `sec` param either.

Reply:
@closetool Thanks for the review.
Sure thing, added `sec` param in https://github.com/casbin/casbin/pull/776/commits/3452597d96873b48e5a9073732602b4103b72221

Reply:
@closetool why adding sec? Does a rule have sec? I think only model file has sec.

Reply:
@hsluoyz for other interface has `sec`, like https://github.com/casbin/casbin/blob/05b22efb450e2e58dd21d31f502261cf46d80f51/persist/adapter.go#L55,
should we keep consistent?

Reply:
OK.